src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -1,26 +1,64 @@
-﻿using System;
+using System;
 using CoreGraphics;
 using UIKit;
+using Microsoft.Maui.Platform.iOS;
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class EditorHandler : AbstractViewHandler<IEditor, UITextView>
+	public partial class EditorHandler : AbstractViewHandler<IEditor, MauiTextView>
 	{
 		static readonly int BaseHeight = 30;
+	
+		static readonly UIColor DefaultPlaceholderColor = ColorExtensions.PlaceholderColor;
 
-		protected override UITextView CreateNativeView()
+		protected override MauiTextView CreateNativeView()
 		{
-			return new UITextView(CGRect.Empty);
+			return new MauiTextView(CGRect.Empty);
+		}
+
+		protected override void SetupDefaults(MauiTextView nativeView) 
+		{
+			nativeView.PlaceholderTextColor = DefaultPlaceholderColor;
```

Comment:
I think this goes the other way around. The defaults are copied from the native view so we can reset them later. This is ever only run once in the lifetime of the handler. Since the text view does not have a placeholder, we don't need this at all and we can just use our own value each time.

Reply:
If you are wanting to set this, then the `CreateNativeView` is the better place.

Reply:
Moved.